packages/astro/test/fixtures/css-order-dynamic-import/src/pages/one.astro

```diff
@@ -1,7 +1,7 @@
 ---
 import '../components/One.astro';
 import '../components/Two.astro';
-await import('../components/Three.astro');
+await import('../components/Three.js');
```

Comment:
This PR caused this test to fail because the `Three.astro` component was imported but it's never used, hence the scoped styles in `Three.astro` weren't injected. Previously it was testing if the scoped styles were injected which I don't think it's quite right.

This test comes from https://github.com/withastro/astro/pull/6176, so to stay true to what it's testing, I changed it to a `.js` file so that the CSS is no longer scoped.